instance/models/instance.py

```diff
@@ -234,6 +235,15 @@ def github_base_url(self):
         return 'https://github.com/{0.fork_name}'.format(self)
 
     @property
+    def github_pr_url(self):
+        """
+        Web URL of the Github PR, or None if the PR number is not set.
+        """
+        if self.github_pr_number is None:
+            return None
+        return '{0.github_base_url}/pull/{0.github_pr_number}'.format(self)
```

Comment:
Oh, darn, of course.  Yes, adding the full URL seems like the best solution until we have some more complete PR model.